src/server/controllers/downloadController.js

```diff
@@ -1,6 +1,18 @@
+const fs = require('fs');
+
+const indexFilePath = "./src/dataBase/file-indexData.json";
+
 module.exports = function (req, res) {
-    const filePath = "src/outputData/outputData.json";
-    const fileName = "outputData.json";
+    const getFileIndex = require('../services/helpers/readFileIndex');
```

Comment:
move this line below `const fs = require('fs');`

Reply:
I thought it should be redefined every time I call it?